src/lib/useFileStore.tsx

```diff
@@ -52,7 +52,9 @@ export const useFileStore = create(
             includeInCommand: true,
             folder: folders.slice(0, folders.length - 1).join("/"),
             hasIncludeDirectives: content.includes("#include"),
-            isEntrypoint: /\(\)\s*(recv_internal|main)\s*\(/.test(content),
+            isEntrypoint:
+              /\(\)\s*(recv_internal|main)\s*\(/.test(content) ||
+              /fun onInternalMessage\s*\(/.test(content),
```

Comment:
@Trinketer22 pls see whether this is the correct test for an entrypoint for .tolk files